code/API_definitions/carrier_billing.yaml

```diff
@@ -181,12 +187,25 @@ paths:
         - Payment
       summary: Get a list of payments
       operationId: retrievePayments
-      description: Retrieve a list of payments and their details based on some filtering criteria.
+      description: | 
+        Retrieve a list of payments and their details based on some filtering criteria.
+        Regardless the payment criteria provided, response MUST be always limited to payments performed by the API client (i.e same oAuth credentials) triggering this request.
+        This is to guarantee no API client can check payments performed by other, therefore avoiding any legal or privacy topic.
+
+        Considerations regarding `startDate`, `endDate`: 
+        - If both included, return payments in that date range
+        - If no one included, no filtering by date range is applied
+        - If only settled `startDate`, `endDate` is considered current date-time
+        - If only settled `endDate`, `startDate` is considered one hour before
       parameters:
         - $ref: "#/components/parameters/x-correlator"
         - $ref: "#/components/parameters/Page"
         - $ref: "#/components/parameters/PerPage"
-        - $ref: "#/components/parameters/Seek"
+        - $ref: "#/components/parameters/StartDate"
```

Comment:
As defined in the guidelines here (https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#83-filtering) should be paymentDate.gt no?

Reply:
Get your point. Two comments:
1) For the matching rule, think is better xxx.gte to include the exact initial date also in the range (in case it happens)
2) I was proposing to refer to paymentCreationDate (maybe we can set it mandatory), because is when a "first" track of the payment is done in the Operator system and we can have ongoing payment processes not completed yet. If we look based on paymentDate we are reducing the scope to the payments already performed, which is a subset of the existing ones. Think that would make sense

Reply:
Agree on both comments :)